src/easylink/graph_components.py

```diff
@@ -237,7 +253,7 @@ def add_edge_from_params(self, edge_params: EdgeParams) -> None:
 
 @dataclass(frozen=True)
 class SlotMapping(ABC):
-    """A mapping between parent and child nodes.
+    """A mapping between a slot on a parent Step and a slot on (one of) its child Steps.
```

Comment:
Are SlotMappings only on StepGraphs (i.e. not on ImplementationGraphs)? If so, should this be explicitly stated here somehow?